packages/flipper-plugin-async-storage-advanced/src/index.tsx

```diff
@@ -38,93 +42,117 @@ const buttonStyle = {
-                </Button>
-              </Layout.Container>
-            </Row>
-          )
-        })
-        }
-      </Layout.Container>
-    </Layout.ScrollContainer >
+                }
+              }}>
+                Add
+              </Button>
+            </Layout.Container>
+
+          </Form>
+        </Layout.Container>
+
+        <Layout.Container style={{ display: 'flex', borderWidth: 1, padding: 20, margin: 10 }}>
+          <Text style={{ marginBottom: 20, fontSize: 18 }}>Local storage items</Text>
+          {data.map((d) => {
+
+            return (
+              <Row style={{ display: 'flex', flexDirection: 'row', marginBottom: 10 }}>
+                <Layout.Container style={{ display: 'flex', flex: 0.3, alignItems: 'center' }}>
+                  <Text style={{ flex: 1 }}>Key: {d.id}</Text>
+                </Layout.Container>
+                <Layout.Container style={{ display: 'flex', flex: 0.6, alignItems: 'center' }}>
+                  <Input
+                    value={d.content}
+                    name={d.id}
+                    onChange={(ev) => {
+                      const indexToUpdate = findIndex(data, (item) => item.id === ev.target.name)
+                      const newData = update(data, {
+                        [indexToUpdate]: {
+                          content: {
+                            $set: ev.target.value
+                          }
+                        }
+                      })
+
+                      instance.setData(newData)
+                    }} />
+                </Layout.Container>
+                <Layout.Container style={{ display: 'flex', flex: 0.6, alignItems: 'center', justifyContent: 'flex-end' }}>
+                  <Button
+                    onClick={() => {
+                      const indexToDelete = findIndex(data, (item) => item.id === d.id)
+                      const newData = update(data, {
+                        $splice: [[indexToDelete, 1]]
+                      })
+
+                      instance.setData(newData)
+                    }}
+                    style={{ marginBottom: 3 }}
+                  >
+                    Delete
+                  </Button>
+                  <Button onClick={() => {
+                    const indexToSelection = findIndex(data, (item) => item.id === d.id)
+                    instance.setSelection(indexToSelection)
+                  }}>
+                    Show it
+                  </Button>
+                </Layout.Container>
+              </Row>
+            )
+          })
+          }
+        </Layout.Container>
+      </Layout.ScrollContainer >
+      <DetailSidebar>
+        {selectedId && renderSidebar(data[selectedId])}
+      </DetailSidebar>
+    </>
   );
 }
+
+function renderSidebar(row: Row) {
+  return (
+    <Layout.Container gap pad>
+      <Typography.Title level={4}>{row.id}</Typography.Title>
+      <DataInspector data={{ ...JSON.parse(row.content) }} expandRoot={true} />
```

Comment:
I am afraid this will crash if the value is different than a stringified object. We need to add some logic around that to make it more bullet proof.

Reply:
The truth is, I usually store objects in storage, but you're right. I will think about it.
It's my first time when i deal with flipper plugin, but it's a very-very exciting thing. I have plenty of ideas how to extends this plugin just like trigger refresh event on storage's changes or adding some advanced editor for values of storage.
But first things first. ;)